site/en/blog/how-fugu-is-my-browser-and-how-fugu-is-the-web/index.md

```diff
@@ -1,11 +1,12 @@
 ---
 layout: 'layouts/blog-post.njk'
-title: '"How Fugu is my browser?" and "How Fugu is the web?"'
+title: 'Discovering the capable web'
 subhead: >
-  To answer these questions, check out a browser testing site and a browser extension, both focused
-  on the web's capabilities.
-date: 2023-01-16
-# updated: 2023-01-16
+  What advanced web capabilities does your browser support? And what are the web apps that make use
+  of these capabilities? To answer these questions, check out a browser testing site and a browser
+  extension that I have built.
```

Comment:
```suggestion
  What advanced web capabilities does your browser support? And what are the web apps that make use
  of these capabilities? To answer these questions, check out a browser testing site and a browser
  extension.
```

Reply:
the "I have built" seems a bit at odds with other content. I'd suggest putting your ownership on the projects themselves.